### ISSUE TITLE
Build is failing

### ISSUE THREAD

**Issue by wreiske** [jonashackt/vagrant-ansible-on-appveyor#1]
Your current build is failing and I am having no luck getting a 64bit OS to work on AppVeyor with virtualbox.

I have tried changing the nictype to Am79C973 without any luck. I think the problem is that there is no virtualization support for the x64 on AppVeyor since it is running inside an LXC.

Any ideas?

**Comment by jonashackt** [comment]
No, I'am really sorry - but I also tried all those options you mentioned (besides 64Bit won't work in the free tier of AppVeyor, since then you need VT-X enabled - which wasn't available for free when I used AppVeyor the last time. So you have to go with 32Bit).

My last try was to contact AppVeyor support, see https://help.appveyor.com/discussions/problems/18217-vagrant-on-appveyor-not-working-both-on-windows-ubuntu - end they also didn't get it to work either. So [this old link](https://help.appveyor.com/discussions/problems/1247-vagrant-not-working-inside-appveyor), which got me on the Vagrant/VirtualBox on AppVeyor trip doesn't seem to tell the truth anymore 😢 

If you have any ideas, PRs are always welcome. For now, I buried my plan and now use AWS together with TravisCI to run my VMs in free tiers. If you're interested, here's an example project (using Ansible & Molecule, which uses Python library boto for the AWS connectivity and a properly configured AWS CLI: https://github.com/jonashackt/molecule-ansible-docker-vagrant

**Comment by jonashackt** [comment]
According to https://help.appveyor.com/discussions/problems/22269-packer-build-fails-on-vboxmanage-error-vboxmanage-error-vt-x-is-not-available-verr_vmx_no_vmx and the test project https://github.com/appveyor-tests/win2k19-azure with it's build output https://ci.appveyor.com/project/appveyor-tests/win2k19-azure/builds/24024378 saying:

```
AppVeyor Azure VMs for heterogeneous build scenarios.
Default is Standard_D2s_v3 VM Size. Custom plan with Standard_D4s_v3 can be created per request.
It is in Preview now. Please send email to support@appveyor.com if you want to try.
```

there's an image `Windows Server 2019` which may supports nested virtualization (VT-x) which is needed to successfully run Vagrant / VirtualBox on AppVeyor. I filed an issue on the appveyor support at https://help.appveyor.com/discussions/questions/46774-nested-virtualization-needed-for-opensource-projects-vt-x-is-not-available-verr_vmx_no_vmx and also send an email to request the Windows Server 2019 image, which is sadly not available right now out-of-the-box (see this build events https://ci.appveyor.com/project/quakefrog/vagrant-ansible-on-appveyor/builds/31012490/events). Let's see, what the AppVeyor support is telling us :)

**Comment by jonashackt** [comment]
Ok @wreiske I don't know if AppVeyor folks are going to help me - but this solution would be somehow nasty anyway, since one has to ask for getting this special image... But I stumbled upon another solution that finally worked 100%: https://github.com/jonashackt/vagrant-travisci-libvrt Try it out, it's based on libvirt & KVM and works like a charm on TravisCI.

**Comment by wreiske** [comment]
Awesome! I'll take a look. Thanks for looking! If only I could remember why I needed this? 🤔 I've slept since then... hah!

EDIT: Now I remember! https://github.com/mieweb/vagrant-ccda-validator
